### DISCUSSION TITLE
Flowise Updated (New Assistant Not Displaying)

### DISCUSSION THREAD

**Discussion by mdwoicke** [FlowiseAI/Flowise#1199] [Q&A] [answered]
Hey guys. I'm having a problem with not seeing the new Assistants in the left nav after updating Git from the latest code release. I can see all the new files come across but not seeing the new functionality of the new Assistants menu or even the new Assistant in the Agent list. I updated my OpenAI package to the latest as well to make sure I had the latest. I have never come across this issue in the dozens of times I've updated my code base and performed a yarn install and build. I even created a new code instance and tried installing with npm. Has anyone else ran into this issue? Any help is much appreciated! Thanks!

**Comment by cycloner2020** [comment, marked as answer]
I just notice the latest build this morning fixed this issue. Its working now.
